integration_tests/integration/select-teams.spec.ts

```diff
@@ -6,27 +6,33 @@ context('Select teams', () => {
     cy.task('reset')
     cy.task('stubSignIn')
     cy.task('stubAuthUser')
+    cy.task('stubGetUnallocatedCasesByTeams', {
```

Comment:
Seems a bit unwieldy in the test code, but I appreciate the return value is complex